.eslintignore

```diff
@@ -5,6 +5,7 @@ dist
 **/wip/*
 node_modules/
 *.min.js
+*.md
```

Comment:
Markdowns have never been linted - the old version of ocular-dev-tools had a bug. Ignoring them for now to avoid breaking lint. I can fix this in a separate PR, which will likely involve some doc format changes.

Reply:
+1